src/utils/mapInstance.js

```diff
@@ -0,0 +1,36 @@
+const DEFAULT_MAP_OPTIONS = { center: { lat: 0, lng: 0 }, zoom: 2 };
```

Comment:
I believe we are keeping it this way for testing? In production this file would be in locus-dashboard-v2 and googleMapInstance would be passed as prop to the ReactGoogleMap component.

Reply:
Removed util from this repo